code/datums/traits/good.dm

```diff
@@ -69,6 +70,28 @@
 
 
 
+/datum/quirk/musician
+	name = "Musician"
+	desc = "You can tune handheld musical instruments to play melodies that clear certain negative effects and soothe the soul."
+	value = 1
+	mob_trait = TRAIT_MUSICIAN
+	gain_text = "<span class='notice'>You know everything about musical instruments.</span>"
+	lose_text = "<span class='danger'>You forget how musical instruments work.</span>"
+
+/datum/quirk/musician/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+
+	var/obj/item/instrument/guitar/guitar = new(get_turf(H))
+	H.put_in_hands(guitar)
+	H.equip_to_slot(guitar, SLOT_IN_BACKPACK)
+
+	var/obj/item/musicaltuner/musicaltuner = new(get_turf(H))
```

Comment:
Not to bother too much, but why not just make this a passive effect they have? Give instruments a signal for when they start and stop playing, and have the quirk listen for it. Of course, if that's too much effort, you could always make it an action button ability they have - The only reason I say this is because it could get really annoying if they lose their tuner item.

Reply:
big agreement with this, also it makes more sense that the musician is better at playing music, rather than being really good at tuning a musicial instrument, but no better at playing it than any other crewmember.

Reply:
Current method allows for more interesting interactions (including losing/getting your tuner stolen) so I'm not really interested in streamlining it and lose that. Protect your tuner like it's a family heirloom!